### PR TITLE
Feat add split at edit cursor

### PR DESCRIPTION
**Pull request by elgiovanno**
Hi! Let me know if I did this correctly :)

### CODE REVIEW

File: definitions/actions.lua [hunk 1]
```diff
@@ -424,6 +424,7 @@ return {
       SnapshotsSaveCurrent = "_SWSSNAPSHOT_SAVE",
       SplitAndSelectItemsInRegion = "_S&M_SPLIT11",
       SplitAtTimeSelection = 40061,
+      SplitItemsAtEditCursor = {40289, {"_XENAKIOS_SELITEMSUNDEDCURSELTX"},40757 ,40289},
```
[the remark this answers is not in the document]
Comment (elgiovanno) [reply definitions/actions.lua:427]: Agreed! Those are "unselect all items" and "split items at edit cursor" I'll check if they don't already exist and add them in if they don't.

[the remark this answers is not in the document]
Comment (elgiovanno) [reply definitions/actions.lua]: Just pushed my new changes. Do I re-request review or is it enough to just push my branch?

[the remark this answers is not in the document]
Comment (gwatcha) [reply definitions/actions.lua]: Just push new changes

File: definitions/actions.lua [hunk 2]
```diff
@@ -424,7 +424,8 @@ return {
       SnapshotsSaveCurrent = "_SWSSNAPSHOT_SAVE",
       SplitAndSelectItemsInRegion = "_S&M_SPLIT11",
       SplitAtTimeSelection = 40061,
-      SplitItemsAtEditCursor = {40289, {"_XENAKIOS_SELITEMSUNDEDCURSELTX"},40757 ,40289},
+      SplitItemsAtEditCursor = {"UnselectItems", "SelectItemsUnderEditCursor", "SplitItemsUnderEditCursor","UnselectItems"},
```
Comment (gwatcha) [inline definitions/actions.lua:427]: Thanks! But to be meticulous... you're missing a space `...r","Uns...`

Comment (gwatcha) [reply definitions/actions.lua:427]: Just do a quick commit to fix that and i'll merge this right away

Comment (elgiovanno) [reply definitions/actions.lua:427]: Gah! Will do

**Comment by elgiovanno** [comment]
That's better!

**Comment by gwatcha** [comment]
thanks!

**Comment by elgiovanno** [comment]
Dammit!!! I just closed this by mistake

**Pull request closed without merging**